packages/evolution-frontend/src/components/inputs/__tests__/InputString.test.tsx

```diff
@@ -32,6 +32,7 @@ test('Should correctly render InputString with all parameters', () =>{
         datatype: 'string' as const,
         maxLength: 20,
         defaultValue: 'test',
+        textTransform: 'lowercase',
```

Comment:
To have the test pass, here you should put `'lowercase' as const` to avoid the type being `string`